files/01_setup_http_service.sh

```diff
@@ -64,6 +88,12 @@ if [[ $rc -ne 0 ]]; then
   ipa service-add-principal "HTTP/$hostname" "HTTP/ip-${ip_address_dashes}.ec2.internal"
 fi
 
+# If an IPA-GROUP-ALLOWED-TO-CREATE-KEYTAB argument was provided,
+# add that group to the service.
+if [ $# -eq 1 ]; then
```

Comment:
I would rather check that the variable has been set than rely on argument count.

```suggestion
if [ -v  group_allowed_to_create_keytab ]; then
```